src/main/java/org/folio/dew/batch/bursarfeesfines/AggregatedAccountReader.java

```diff
@@ -101,7 +105,7 @@ public void initStep(StepExecution stepExecution) {
 
       userToAccountsListMap.computeIfAbsent(
         user,
-        (User key) -> new ArrayList<Account>(Arrays.asList(account))
+        (User key) -> new ArrayList<Account>()
       );
 
       userToAccountsListMap.computeIfPresent(
```

Comment:
missed this on the other PR. you can simplify this a bit with:

```java
      userToAccountsListMap.computeIfAbsent(
        user,
        (User key) -> new ArrayList<Account>()
      ).add(account);
```

The `computeIfAbsent` will return the current (or computed) value; then, since `ArrayList` is mutable in place, you can just call `.add` on it (no need to save it back in)